maint/release.py

```diff
@@ -0,0 +1,104 @@
+#! /usr/bin/env python3
+##
+## Copyright (C) by Argonne National Laboratory
+##     See COPYRIGHT in top-level directory
+##
+
+import sys
+import argparse
+import tempfile
+import os
+import subprocess
+import re
+from datetime import datetime
+
+class colors:
+    FAILURE = '\033[1;31m' # red
+    SUCCESS = '\033[1;32m' # green
+    INFO    = '\033[1;33m' # yellow
+    PREFIX  = '\033[1;36m' # cyan
+    OTHER   = '\033[1;35m' # purple
+    END     = '\033[0m'    # reset
+
+
+##### run_cmd function
+def runcmd(runargs, critical=True, silent=False):
+    sys.stdout.write(colors.PREFIX + ">>>>> " + colors.END)
+    for x in runargs:
+        sys.stdout.write(x + " ")
+    sys.stdout.write("\n")
+    p = subprocess.Popen(runargs, stdout = subprocess.PIPE, stderr = subprocess.STDOUT)
+    ret = p.wait()
+    out = p.communicate()[0].decode().strip()
+    if (ret != 0):
+        if (critical):
+            print(colors.FAILURE + "==== command failed (critical) ====\n" + colors.END)
+            print(out)
+            print(colors.FAILURE + "==== command output complete ====\n" + colors.END)
+            sys.exit()
+        else:
+            print(colors.FAILURE + "==== command failed (warning) ====\n" + colors.END)
+            print(out)
+            print(colors.FAILURE + "==== command output complete ====\n" + colors.END)
+            sys.exit()
+    return out
+
+
+##### main function
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--version', help='tarball version to create', required=True)
+    args = parser.parse_args()
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+        oldpwd = os.getcwd()
+
+        # go to the tmpdir
+        os.chdir(tmpdir)
+
+        # checkout the release tag
+        runcmd(['git', 'clone', 'https://github.com/pavanbalaji/yaksa.git'])
```

Comment:
Is the repo url to be updated to pmodels?

Reply:
Yikes!  Thanks for catching it.  I was using this for debugging.